### PR TITLE
Setup airbrake

### PR DESCRIPTION
**Pull request by saikatbhadra**
Sets up airbrake so all javascript uncaught exceptions are logged and sent to the #dev channel.

Closes: https://github.com/findthemasks/findthemasks/issues/462

### CODE REVIEW

File: public/airbrake.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// TODO we should have the environment change so we don't alert on local dev machines
+let environment;
+if (location.hostname === "localhost" || location.hostname === "127.0.0.1") {
```
Comment (nburt) [inline public/airbrake.js:3]: what about local.findthemasks.com?

File: public/give.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
   gtag('js', new Date());
   gtag('config', 'UA-161442455-1');
   </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/airbrake-js/1.6.7/client.min.js"></script>
```
Comment (nburt) [inline public/give.html:12]: should we also add to stats / whoweare? Stats has some js, I don't believe whoweare does

Comment (crayne) [reply public/give.html:12]: Maybe there should be a separate channel for this.

File: public/airbrake.js (hunk 1 above)
Comment (saikatbhadra) [reply public/airbrake.js]: Done

File: public/give.html (hunk 2 above)
Comment (saikatbhadra) [reply public/give.html:12]: We need to put all the critical js in a common js that is included in all the pages. We shouldn't have to remember to include it on each page (esp as I anticipate the new UX to give us more pages).

**Pull request merged**